rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,148 @@
+  - [BioStudies](https://www.ebi.ac.uk/biostudies/)
+  - [Zenodo](https://en.wikipedia.org/wiki/Zenodo)
+  - [Figshare](https://en.wikipedia.org/wiki/Figshare)
+  - [Crossref](https://en.wikipedia.org/wiki/Crossref)
+  - HCA DCP assigns its own DOIs by becoming a member of the International DOI Foundation (IDF)
+
+#### DOI Versioning
+Most open access repositories that provide DOI minting services appear to provide support for versioning (i.e. multiple versions of a single citation available e.g. via a drop-down selection), the ability to store accompanying files in the repository (this would be useful for storing versions of manifest and metadata tsv files, etc.), and the ability to store URL references (useful for linking back to a page in the Data Browser).
+Examining how both Figshare and Zenodo implement DOIs, as the group minting DOIs controls everything after the prefix, it is possible to have a main DOI which points to the most recent version and then versioned URLs which point to specific versions.
+
+#### Data Browser support for DOIs
+The Data Browser could contain a reference to the base DOI for each project. Clicking on the DOI link would then redirect the user to the (external) DOI repository where the user could see all the versions of that project and download the associated manifest and metadata.tsv files for a specific version. 
+
+#### DOI Creation/Update Process
+Embedding a DOI in the metadata (e.g. biostudies_doi) during ingest would provide per-project citability.
+Provided the DOI repository supports versioning the metadata DOI field could be updated automatically by Ingest to a new version whenever a project update is processed.
+Using a DOI that links to an external repository that can store a manifest for each version of a project may be the simplest way to provide access to versions of the data for a project.
+
+The creation/update process would perform the following steps:
+  - Ingest creates a new DOI (new project) or a new version of an existing DOI (updated project) when the submission is deemed “complete”.
+  - Upload the corresponding metadata tsv and manifest file for this version of the project to the DOI repository (need to wait for these to be generated).
+  - Upload the matrix output file(s) for this version of the project to the DOI repository.
+  - Update a project description to the DOI repository (e.g. what’s in this version?).
+  - Create a link in the DOI repository back to the project details page in the Data Browser.
+
+### Acceptance Criteria [optional]
+
+#### Phase 1
+  - A citation link is provided in the Data Browser for each project
+  - The citation link remains valid for the lifetime of the DCP
+  - The citation link is a URL which resolves to the latest version of the project details page for the specified project
+  - The citation URL is used in publications by scientists to cite DCP project data
```

Comment:
This is a great statement but it is a success metric (which we should track) rather than an acceptance criterion. This won't be true when the feature is released but we should make sure we measure how frequently people cite us so we can understand the success of the feature

Reply:
Removed "used in publications" from acceptance criteria.